boards/shields/dfrobot_can_bus_v2_0/dfrobot_can_bus_v2_0.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2019 Nick Ward
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_spi {
+	status = "okay";
+	cs-gpios = <&arduino_header 16 0>; /* D10 */
+
+	mcp2515@0 {
+		compatible = "microchip,mcp2515";
+		spi-max-frequency = <1000000>;
+		int-gpios = <&arduino_header 8 0>; /* D2 */
+		status = "okay";
+		label = "CAN_1";
+		reg = <0x0>;
+		bus-speed = <250000>;
```

Comment:
Please use 125000 here. We agreed to have a common speed for all boards.

Reply:
No problem.  For my future reference would you have a link for this agreement and any others?  Thanks

Reply:
@alexanderwachter  I've updated the bus-speed to 125000 and (if all ok) added two commits to fix this issue in other MCP2515 DTS files and one spot in the CAN driver sample which was also using 250000.

Reply:
Thanks for fixing the sample too. This "agreement" was made after I noticed that the boards are not compatible with each other. It's not written down somewhere.